felles-db/src/main/kotlin/no/nav/helsearbeidsgiver/felles/db/Database.kt

```diff
@@ -0,0 +1,65 @@
+package no.nav.helsearbeidsgiver.felles.db
+
+import com.zaxxer.hikari.HikariConfig
+import com.zaxxer.hikari.HikariDataSource
+import no.nav.helsearbeidsgiver.felles.fromEnv
+import org.flywaydb.core.Flyway
+import org.jetbrains.exposed.sql.Database as ExposedDatabase
+
+class Database(
+    private val config: HikariConfig
+) {
+    constructor(secrets: Secrets) : this(
+        dbConfig(secrets)
+    )
+
+    val dataSource by lazy { HikariDataSource(config) }
+    val db by lazy { ExposedDatabase.connect(dataSource) }
```

Comment:
Ikke noe big deal, men nå dikterer felles-db bruk av Exposed, om man vil ha "felles felles" for alle dber, så kunne dette kanskje bli droppet her, eller vært mulig å toggle / unngå.

Reply:
Men alle bruker Exposed i dag? Jeg er enig i at lock-in ikke er optimalt, men tenker at det ikke er så mye kode at man ikke kan inline det om man evt. går bort i fra Exposed i fremtiden.

Hadde det bare vært den ene linja så hadde jeg lett endret det, men så er det noe i testene også som gjør at man ville spredd noe boilerplate og gjort koden hakket vanskeligere å lese.

Reply:
Hmm, det hadde jeg ikke fått med meg 🤔 Da bør felles-db kanskje omdøpes til exposed-db, så kan man bruke det der man vil ha Exposed.